tests/openmetrics/test_parser.py

```diff
@@ -252,7 +252,7 @@ def test_labels_with_invalid_utf8_values(self):
             if sys.version_info >= (3,):
                 inj = u'\uD802'
             else:
-                inj = '\xf8\xa1\xa1\xa1\xa1'
+                inj = '\xfc'
```

Comment:
\xff should do it

Reply:
so using this or any in this way dint help but using them as a binary string seemed to have done the trick